benchmarks/shootout/build_iwasm_sgx.sh

```diff
@@ -1,38 +1,45 @@
 # build different running modes for iwasm 
-mkdir -p out
+cp -r ${WAMR_DIR}/product-mini/platforms/linux-sgx  ${WAMR_DIR}/product-mini/platforms/linux-sgx-classic-interp
+cp -r ${WAMR_DIR}/product-mini/platforms/linux-sgx  ${WAMR_DIR}/product-mini/platforms/linux-sgx-fast-interp
+cp -r ${WAMR_DIR}/product-mini/platforms/linux-sgx  ${WAMR_DIR}/product-mini/platforms/linux-sgx-aot
+cp -r ${WAMR_DIR}/product-mini/platforms/linux-sgx  ${WAMR_DIR}/product-mini/platforms/linux-sgx-fast-jit
 
-cp -r ${WAMR_DIR}/product-mini/platforms/linux-sgx  ${WAMR_DIR}/product-mini/platforms/linux-sgx-1
-cp -r ${WAMR_DIR}/product-mini/platforms/linux-sgx  ${WAMR_DIR}/product-mini/platforms/linux-sgx-2
-cp -r ${WAMR_DIR}/product-mini/platforms/linux-sgx  ${WAMR_DIR}/product-mini/platforms/linux-sgx-3
+cd ${WAMR_DIR}/product-mini/platforms/linux-sgx-classic-interp
+rm -r build
+mkdir build && cd build
+# build iwasm with interpreter
+cmake .. -DWAMR_BUILD_FAST_INTERP=0 -DWAMR_BUILD_AOT=0
+make
+cd ../enclave-sample
+make
 
-# in directory WAMR_DIR
-cd ${WAMR_DIR}/product-mini/platforms/linux-sgx-1
-sudo rm -r build
+cd ${WAMR_DIR}/product-mini/platforms/linux-sgx-fast-interp
+rm -r build
 mkdir build && cd build
-# build iwasm with AOT and interpreter
-cmake .. -DWAMR_BUILD_FAST_INTERP=0
+# build iwasm with fast-interpreter
+cmake .. -DWAMR_BUILD_AOT=0
 make
 cd ../enclave-sample
 make
+cp iwasm iwasm_fast_interp
```

Comment:
Seems no need to copy again, the folder name has taken the mode info
And it doesn't copy for classic-interpreter, after L14